databases/week1/db-week1.sql

```diff
@@ -0,0 +1,84 @@
+/*Find out how many tasks are in the task table*/
+SELECT 
+    COUNT(*) as 'Total Task'
+FROM
+    Task;
+/*Find out how many tasks in the task table do not have a valid due date*/
+SELECT 
+    COUNT(*) as 'unvalid due date tasks'
+FROM
+    Task
+WHERE
+    due_date IS NULL;
+/*Find all the tasks that are marked as done*/
+SELECT 
+    *
+FROM
+    task
+        JOIN
+    status ON task.status_id = status.id
+WHERE
+    name = 'Done';
+/*Find all the tasks that are not marked as done*/
+SELECT 
+    *
+FROM
+    task
+        JOIN
+    status ON task.status_id = status.id
+WHERE
+    name != 'Done';
+/*Get all the tasks, sorted with the most recently created first*/
+SELECT 
+    *
+FROM
+    Task
+ORDER BY created DESC;
+/*Get the single most recently created task*/
+SELECT 
+    *
+FROM
+    Task
+ORDER BY created DESC
+LIMIT 1;
+/*Get the title and due date of all tasks where the title or description contains 'database'*/
+
+SELECT 
+    title, due_date
+FROM
+    task
+WHERE
+    title REGEXP 'database'
+        OR description REGEXP 'database';
```

Comment:
```suggestion
    title LIKE '%database%'
        OR description like '%database%';
```

Reply:
Thank you for your precious time for the review and giving valuable feedback. May be it is wrong but I heard that REGEXP is newer and more powerful than like. Would you please guide me about this confusion?